main.py

```diff
@@ -92,6 +92,8 @@ def check_origin(self, origin):
 
 def main():
     application = tornado.web.Application([
+        (r"/rocon_web_remocon/(.*)",tornado.web.StaticFileHandler,{"path":"./www/rocon_web_remocon", "default_filename":"index.html"}),
+        (r"/rocon_web_common/(.*)",tornado.web.StaticFileHandler,{"path":"./www/rocon_web_common", "default_filename":"index.html"}),
```

Comment:
@pablisho Instead of creating a file handler for each path, can we move the RosbridgeProxyHandler to /ws and put the static file handler in the root context (/)?
This way we can keep adding more subdirectories with static HTML/JS code as we need without the need to register more and more file handlers each time.